File: app/src/main/java/id/rackspira/seedisaster/data/network/entity/ListNewsBencana.java

package id.rackspira.seedisaster.data.network.entity;

import com.google.gson.annotations.SerializedName;

public class ListNewsBencana {

    @SerializedName("href")
    private String mHref;
    @SerializedName("id")
    private String mId;
    @SerializedName("score")
    private Long mScore;

    public String getHref() {
        return mHref;
    }

    public void setHref(String href) {
        mHref = href;
    }

    public String getId() {
        return mId;
    }

    public void setId(String id) {
        mId = id;
    }

    public Long getScore() {
        return mScore;
    }

    public void setScore(Long score) {
        mScore = score;
    }

}

File: app/src/main/java/id/rackspira/seedisaster/data/network/entity/ListNewsBencana.kt
package id.rackspira.seedisaster.data.network.entity

import com.google.gson.annotations.SerializedName

data class ListNewsBencana(

    @SerializedName("href")
    var href: String? = null,
    @SerializedName("id")
    var id: String? = null,
    @SerializedName("score")
    var score: Long? = null,
    @SerializedName("fields")
    var fields: Fields? = null

)
